src/test/java/unit/BqDatasetControlled.java

```diff
@@ -346,6 +346,17 @@ void updateIndividualProperties() throws IOException {
     assertEquals(newName, updateDataset.name);
     assertEquals(newDescription, updateDataset.description);
 
+    // update just the cloningInstructions
+    // `terra resources update bq-dataset --cloning=$CloningInstructionsEnum.DEFINITION`
+    updateDataset =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFBqDataset.class,
+            "resource",
+            "update",
+            "bq-dataset",
+            "--cloning=" + CloningInstructionsEnum.DEFINITION);
+    assertEquals(CloningInstructionsEnum.DEFINITION, updateDataset.cloningInstructions);
```

Comment:
This is just asserting the request, which is not really useful, it's not testing your code. Delete line 358 and add line 364:

```
assertEquals(cloningInstructions, describeDataset.cloningInstructions);
```

Reply:
Hi Melissa. We do not have `cloningInstructions`. do you mean `updatedDataset` ?

Reply:
Sorry, I misunderstood. `updateDataset` is returned by backend, so what you have is fine. Line 358 is fine. For completeness, please add to line 365:

```
assertEquals(CloningInstructionsEnum.DEFINITION, describeDataset.cloningInstructions);
```

It's possible that `terra resource update` returns the right cloning instructions, but `terra resource describe` doesn't.